### ISSUE TITLE
Android things support

### ISSUE THREAD

**Issue by tyczj** [bertrandmartel/fadecandy-android#3]
Any plans to make this compatible with Android things devices?

**Comment by bertrandmartel** [comment]
Hello, I've never thought about Android things, it seems to be a relevant platform for this kind of usecase. I was planning to develop a version for Android TV in september-october. I will look at Android Things in the same time, thank you letting me know

**Comment by tyczj** [comment]
No problem, I have been looking for an easier way to interface with my LED's I have currently loosely connected to my things device (pi device connected to an arduino device with a breadboard). getting rid of a large arduino device with a smaller fadecandy would be great.

I just found your library today so I will see if I can help any along the way

**Comment by tyczj** [comment]
I got a chance to test this and I dont think there is any specific android things support needed. Since communication is over USB all the current logic still applies and works fine

**Comment by bertrandmartel** [comment]
That's great, I was thinking the USB permission would still be needed [here](https://github.com/bertrandmartel/fadecandy-android/blob/master/fadecandy-service/src/main/java/fr/bmartel/android/fadecandy/service/FadecandyService.java#L336-L355) when a new Fadecandy device is plugged in. It would mean USB permission is auto granted on Android things. Maybe that's the case ?

**Comment by tyczj** [comment]
Yeah that prompt that comes out on a normal android device does not come into effect with android things. All permissions are auto granted the caveat being dangerous permissions are only granted after a device reboot
